tests/test_query.py

```diff
@@ -92,6 +92,84 @@ def test_get_request_without_access(client, mock_arborist_requests):
     assert not_found_err == unauthorized_err
 
 
+def test_get_filtered_requests(client):
```

Comment:
the test looks good, could you just change the usernames to make sure this test doesn't assume that the request's username is the same as the user's username

Reply:
Can I just add a `"username": "requestor_user",` field in the request or do we have to setup a different mock user for this test?

Reply:
We shouldn't use the `requestor_user` username since i think that's the logged in user. Do we have any other usernames that are already set up for the tests?

Reply:
I don't think we've got an additional username set up, but I could add one for this test. Naming the username as `requestor_test_user` unless you have a better name in mind. Let me know.

Reply:
how about `other_user` to make it obvious that we don't want to use the default - the difference between "requestor_user" and "requestor_test_user" is easy to miss when we read fast